main/src/main/java/org/mobilitydata/gtfsvalidator/report/model/FeedMetadata.java

```diff
@@ -591,4 +591,13 @@ public void setFilenames(ImmutableSortedSet<String> filenames) {
   public ImmutableSortedSet<String> getFilenames() {
     return filenames;
   }
+
+  public Boolean hasFlexFeatures() {
+    return specFeatures.keySet().stream()
+        .anyMatch(
+            feature ->
+                feature.getFeatureGroup() != null
+                    && feature.getFeatureGroup().equals("Flexible Services")
+                    && specFeatures.get(feature));
```

Comment:
Out of scope for this PR, but I wish specFeatures name was changed to specFeaturesPresence or something like this. The current name is misleading.